### ISSUE TITLE
Moving folders with large files deletes and then reuploads

### ISSUE THREAD

**Issue by davedodo** [owncloud/client#6827]
### Expected behaviour
Moving a folder with large files (500 MiB) inside locally should move the folder on the server

### Actual behaviour
The folder gets deleted and then reuploaded in the new location (but only after restarting the client or modifying the moved folder in any way e.g. creating an empty text file inside)

### Steps to reproduce
1. Create a folder `sync_dir/foo/bar` with large files inside
2. Move the folder `bar` into `sync_dir/baz`

For some reason, it appears to work fine with small files.

### Server configuration
Operating system:
Ubuntu 16.04

Web server: 
Apache 2.4

Database:
MySQL 5.7

PHP version:
7.1.23

ownCloud version:
10.0.10

Storage backend (external storage):
No external storage

### Client configuration
Client version:
2.5.0

Operating system:
Windows 10 x64

OS language:
English (US)

Installation path of client:
Default

### Logs
```10-18 22:31:02:389 [ info sync.csync.updater ]:	 <= Closing walk for X:/Local/SyncFolder/foo/acp/sea1 with read_from_db 0
10-18 22:31:02:389 [ info sync.csync.updater ]:	 <= Closing walk for X:/Local/SyncFolder/foo/acp with read_from_db 0
10-18 22:31:04:661 [ info sync.networkjob.mkcol ]:	MKCOL of QUrl("https://example.org/remote.php/dav/files/dave/foo/acp/sea1") FINISHED WITH STATUS "OK"
10-18 22:31:04:661 [ info sync.database ]:	Updating file record for path: "foo/acp/sea1" inode: 156897 modtime: 1522534618 type: 2 etag: "" fileId: "00186292oceb9473c42f" remotePerm: "" fileSize: 4096 checksum: ""
10-18 22:31:04:665 [ info sync.propagator ]:	Completed propagation of "foo/acp/sea1" by OCC::PropagateRemoteMkdir(0x890b658) with status 4
10-18 22:31:04:666 [ info sync.propagator ]:	Starting INSTRUCTION_RENAME propagation of "counter/acp/sea1/acp_501301.avi" by OCC::PropagateRemoteMove(0x7c56dd8)
10-18 22:31:04:667 [ info sync.accessmanager ]:	6 "MOVE" "https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi" has X-Request-ID "009370e2-7395-4cfc-9a09-87bfc80f2fac"
10-18 22:31:04:667 [ info sync.networkjob ]:	OCC::MoveJob created for "https://example.org" + "/counter/acp/sea1/acp_501301.avi" "OCC::PropagateRemoteMove"
10-18 22:31:04:884 [ warning sync.networkjob ]:	QNetworkReply::NetworkError(ContentNotFoundError) "Server replied \"404 Not Found\" to \"MOVE https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi\"" QVariant(int, 404)
10-18 22:31:04:886 [ info sync.networkjob.move ]:	MOVE of QUrl("https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi") FINISHED WITH STATUS "ContentNotFoundError Server replied \"404 Not Found\" to \"MOVE https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi\""
10-18 22:31:04:887 [ info sync.database ]:	Setting blacklist entry for  "counter/acp/sea1/acp_501301.avi" 1 "Server replied \"404 Not Found\" to \"MOVE https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi\"" 1539894664 25 1515880624 "c80798581c846dcb39a3372205525bba" "foo/acp/sea1/acp_501301.avi" 0
10-18 22:31:04:890 [ warning sync.propagator ]:	Could not complete propagation of "foo/acp/sea1/acp_501301.avi" by OCC::PropagateRemoteMove(0x7c56dd8) with status 2 and error: "Server replied \"404 Not Found\" to \"MOVE https://example.org/remote.php/dav/files/dave/counter/acp/sea1/acp_501301.avi\""
10-18 22:31:04:899 [ info gui.socketapi ]:	Sending SocketAPI message --> "STATUS:SYNC:X:\\Local\\SyncFolder\\foo\\acp" to QLocalSocket(0x701a040)
10-18 22:31:04:900 [ info gui.socketapi ]:	Sending SocketAPI message --> "STATUS:SYNC:X:\\Local\\SyncFolder\\foo\\acp\\sea1" to QLocalSocket(0x701a040)
10-18 22:31:04:934 [ info sync.networkjob.delete ]:	DELETE of QUrl("https://example.org/remote.php/dav/files/dave/counter/acp/sea1") FINISHED WITH STATUS "OK"
10-18 22:31:04:940 [ info sync.propagator ]:	Completed propagation of "counter/acp/sea1" by OCC::PropagateRemoteDelete(0x890b448) with status 4```

**Comment by ogoffart** [comment]
Looks like a duplicate of https://github.com/owncloud/client/issues/6808
Can you try with the 2.5.1 daily build?
https://download.owncloud.com/desktop/daily/

**Comment by ogoffart** [comment]
Assumed to be fixed in 2.5.1
